platform/security/core/security-core-services/src/main/java/ddf/security/soap/impl/SecureProxyServiceFactoryImpl.java

```diff
@@ -326,7 +326,7 @@ public void handleMessage(Message message) throws Fault {
       }
 
       Collection<AssertionInfo> ais = getAllAssertionsByLocalname(aim, SPConstants.ISSUED_TOKEN);
-      if (ais == null) {
+      if (ais == null || ais.isEmpty()) {
```

Comment:
Update unit tests?

Reply:
There are no unit tests to update for this 😕